www/src/util/config.ts

```diff
@@ -3,6 +3,7 @@ const AUTH0_CLIENT_ID = process.env.REACT_APP_AUTH0_CLIENT_ID || '';
 const SERVER_ENDPOINT = process.env.REACT_APP_SERVER_ENDPOINT || 'http://localhost:1337';
 const SERVER_AUDIENCE = process.env.REACT_APP_SERVER_AUDIENCE || '';
 const MAILCHIMP_URL = process.env.REACT_APP_MAILCHIMP_URL || '';
+const ADOBE_PDF_ID = process.env.ADOBE_PDF_ID || '';
```

Comment:
```suggestion
const ADOBE_PDF_ID = process.env.REACT_APP_ADOBE_PDF_ID || '';
```
And can you update the `.env.sample` file in `www` to include `MAILCHIMP_URL` and `ADOBE_PDF_ID`? Thanks!

Reply:
Good catch on these, made the modifications